include/constants/pokemon.h

```diff
@@ -293,7 +293,8 @@
 #define F_SUMMARY_SCREEN_FLIP_SPRITE 0x80
 
 // Evolution types
-#define EVO_MEGA_EVOLUTION   0xffff // Not an actual evolution, used to temporarily mega evolve in battle.
+#define EVO_MEGA_EVOLUTION      0xffff // Not an actual evolution, used to temporarily mega evolve in battle.
+#define EVO_WISH_MEGA_EVOLUTION 0xfffe // Not an actual evolution, used to temporarily mega evolve in battle.
```

Comment:
perhaps the name Move Mega Evolution instead of Wish Mega Evolution would be better, because the mechanic has nothing to do with wishes and all to do with moves (not forcing you to change it, just wanted to throw the idea out there)

Reply:
I figured it would be Wish Mega Evolution as it's the only instance of it in the series and the battle message references a wish as opposed to an item.

Reply:
Generally I prefer a more generic name just in case (for whatever reason) someone decides to add more, and I feel like the name should describe the mechanic more than the battle message

Reply:
To be fair, leaving it as "Wish Mega Evolution" while fitting for Rayquaza, is not so fitting in the case that a user wanted to repurpose that type of Mega Evolution for a different species.

I agree with Expo here, having a more objective label for the mechanic is a good idea.